warp/Network/Wai/Handler/Warp.hs

```diff
@@ -436,6 +437,13 @@ setGracefulShutdownTimeout :: Maybe Int
                            -> Settings -> Settings
 setGracefulShutdownTimeout time y = y { settingsGracefulShutdownTimeout = time }
 
+-- | Set the maximum header size that Warp will tolerate.
```

Comment:
This should say that this is effective only for HTTP/1.x, not for HTTP/2.

Reply:
Fixed.